css/sidebar.css

```diff
@@ -0,0 +1,69 @@
+.sidebar{
+    grid-area: sidebar;
+    position: sticky;
+    top: 0rem;
+    height: 100vh;
+    overflow-y: auto;
+    max-width: 225px;
+    color: black;
+}
+
+.sb-group{
+    padding: 0.5rem 1rem 0;
+}
+
+.sb-header{
+    border-bottom: 1px solid rgba(65, 65, 65, 0.61);
+    padding: 0.4rem 0 0.4rem 1rem;
+    font-weight: bold;
+}
+
+.sb-item{
+    color: black;
+    cursor: pointer;
+    display: block;
+    text-decoration: none;
+    padding: 0.4rem 0 0.4rem 1rem;
+    margin: 0.5rem 0;
+    border-radius: 4px;
+}
+
+.sb-item.active{
+    background: var(--secondary);
+}
+
+.sb-item:hover{
+    background: var(--secondary);
+}
+
+
+.sidebar-components{
+    color: var(--primary);
+    list-style: none;
+    height: 100%;
+}
+
+.sidebar-item{
```

Comment:
are `sb-item` and `sidebar-item` different?

Reply:
`sidebar-item` was the previous naming convention. Forgot to remove it.